### ISSUE TITLE
`gl_hints` test fails on my system

### ISSUE THREAD

**Issue by dromer** [lv2/pugl#63]
Debian 11 with an NVIDIA RTX2060 card. `dpf-v2` branch:

```
test_gl_hints: ../test/test_gl_hints.c:92: main: Assertion `puglGetViewHint(view, PUGL_SWAP_INTERVAL) != PUGL_DONT_CARE' failed.
```

Commenting out this one assertion completes everything successfully.

**Comment by falkTX** [comment]
Does it happen with the original pugl?

**Comment by dromer** [comment]
Heh, actually nope. Also fails on this step.

**Comment by drobilla** [comment]
Seems like your driver might have a broken implementation of `GLX_EXT_swap_control` and/or pugl is broken for not explicitly checking the extension string.

Try https://github.com/lv2/pugl/tree/get-swap-interval ?

**Comment by dromer** [comment]
@drobilla on that branch all the tests (17) succeed

**Comment by drobilla** [comment]
@dromer Aha.  Great, thanks.

So, for the record, this NVidia driver does implement `GLX_EXT_swap_control`, but `glXQueryDrawable` only works when the context is active (unlike other drivers where it works outside as well).

The potentially user-visible effect of this would be applications not being able to tell if vsync is enabled.  That should work now.

**Comment by dromer** [comment]
Just for posterity the driver I'm currently using is `460.91.03`

**Comment by drobilla** [comment]
Fixed in cea2a9b9
